crates/driver/src/infra/blockchain/gas.rs

```diff
@@ -42,9 +43,16 @@ impl GasPriceEstimator {
                     mempool.additional_tip_percentage,
                 )
             });
+        // Use the lowest max_fee_per_gas of all mempools as the max_fee_per_gas
```

Comment:
Either the comment is wrong or you should be using `.min()`, no?
```suggestion
        // Use the highest max_fee_per_gas of all mempools as the max_fee_per_gas
```